column_computation.hh

```diff
@@ -54,10 +54,19 @@ public:
     virtual bytes_opt compute_value(const schema& schema, const partition_key& key, const clustering_row& row) const = 0;
 };
 
-class token_column_computation : public column_computation {
+/*
+ * Computes token value of partition key and returns it as bytes.
+ *
+ * Should NOT be used, because ordering
+ * of bytes is different than ordering of tokens (signed vs unsigned comparison).
+ *
+ * Not referred to as particular version in schema, due to
```

Comment:
I didn't understand this sentence. Only on further reading, I understood you meant that it is not used in new tables.